keras_cv/models/object_detection/yolox/__internal__/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
sorry for the churn in this ---

can we get rid of __internal__ and just have these under the `yolox` directory?  

we're migrating to a generated API so we will be getting rid of the __internal__ structure